### PR TITLE
Refactoring owner panel

### PR DESCRIPTION
**Pull request by Zeicks**

### CODE REVIEW

File: src/main/java/com/easyrest/components/BaseComponent.java [hunk 1]
```diff
@@ -14,5 +16,8 @@ protected void enterData(String data, By element, WebDriver driver){
         driver.findElement(element).clear();
         driver.findElement(element).sendKeys(data);
     }
-
+    protected void clearWithKeysAndFillInput(By element, String inputText){
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/BaseComponent.java]: explain please how this method work?

File: src/main/java/com/easyrest/components/TagsMenu.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    private final By burgersTag = By.xpath("//li [@data-value = 'burgers']");
+    private final By cuisineJapaneseTag = By.xpath("//li [@data-value = 'japanese cuisine']");
+    private final By greelTag = By.xpath("//li [@data-value = 'greel']");
+    private final By coffeTeaTag = By.xpath("//li [@data-value = 'coffe/tea']");
+
+    public TagsMenu(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void clickOnTagBeer() {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/TagsMenu.java]: maybe should return object to build chain?

File: src/main/java/com/easyrest/components/owner/Details.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package com.easyrest.components.owner;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Details {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/owner/Details.java:6]: Is it restaurant details? Maybe named more specifically?

File: src/main/java/com/easyrest/components/owner/Details.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    private WebDriver driver;
+
+    private final By editInformationButton = By.xpath("//button [@title = 'Edit Information']");
+    private final By updateButton = By.xpath("//span [text() = 'Update']");
+    private final By cancelButton = By.xpath("//span [text() = 'Cancel']");
+
+    public Details (WebDriver driver) {this.driver = driver;}
+
+    public void clickOnButtonEditInformation() {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/owner/Details.java:15]: after click we stay on the same page or go to another?

File: src/main/java/com/easyrest/components/BaseComponent.java (hunk 1 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/BaseComponent.java]: This method highlights the line, clears it and inserts the text

File: src/main/java/com/easyrest/components/owner/Waiters.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+    public Waiters(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void clickOnButtonDeleteWaiter() {
+        driver.findElement(deleteWaiterButton).click();
+    }
+
+    public void clickOnButtonAddWaiter() {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/owner/Waiters.java]: should this method return something?

File: src/main/java/com/easyrest/components/owner/Details.java (hunk 3 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/Details.java:6]: image

File: src/main/java/com/easyrest/components/owner/Details.java (hunk 4 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/Details.java:15]: in this page

File: src/main/java/com/easyrest/components/TagsMenu.java (hunk 2 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/TagsMenu.java]: Good idea

File: src/main/java/com/easyrest/components/owner/Waiters.java (hunk 5 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/Waiters.java]: I don't think it's necessary

File: src/main/java/com/easyrest/components/owner/Details.java (hunk 3 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/Details.java:6]: All components are in one package, I think there should be no problems with understanding

**Review by ViktoriaMilkevich — APPROVED** [review]

**Review by ViktoriaMilkevich — APPROVED** [review]

**Review by martadiachok — APPROVED** [review]

**Review by vadymtsehelny — APPROVED** [review]

**Pull request closed without merging**